content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -13,6 +13,11 @@ const NewTabPage = React.createClass({
   onSearch(value) {
     this.props.dispatch(actions.NotifyPerformSearch(value));
   },
+  componentWillUpdate(nextProps, nextState) {
+    if (nextProps.isReady) {
+      this.props.dispatch(actions.NotifyTelemetry("NEWTAB_RENDER"));
```

Comment:
changing to componentDidUpdate() , this will send the event when newtab page is ready